.azure-pipelines/sync-fork.yml

```diff
@@ -0,0 +1,54 @@
+parameters:
+  - name: fork_repo_url
+    displayName: fork_repo_url
+    default: https://github.com/microsoft/openjdk-jdk
+  - name: upstream_repo_url
+    displayName: upstream_repo_url
+    default: https://github.com/openjdk/jdk
+  - name: fork_branch
+    default: main
+    displayName: fork_branch
+  - name: upstream_branch
+    default: master 
+    displayName: upstream_branch
+
+trigger: none
+
+schedules:
+  # At 5am UTC (10pm PST) daily
+  - cron: "0 5 * * *"
+    displayName: Sync
+    always: false # only run if there has been code changes to the jdk codebase
+    branches:
+      include:
+        - "*"
+stages:
+- stage: sync
+  jobs:
+  - job: sync
+    displayName: "sync repository with upstream"
+
+    pool:
+      name: JEG-ubuntu20.04-x64-EO
+    
+    steps:    
+      - bash: |
+          fork_repo_url=${{ parameters.fork_repo_url }}
+          repourl="${fork_repo_url#https://github.com/schandra13/}"
```

Comment:
What can you do here to determine the repo name instead of hard-coding the expected repo base URL here? What delimiters can you think of using to pull apart a GitHub URL that would give you this information?

Simple string manipulation in `bash` is generally ok, but it's not obvious to people that don't program in `bash` frequently as to what you are trying to accomplish. Be sure to comment lines like this one to make it obvious what you are trying to achieve. (What does the `#` operator do here, for instance).

Reply:
Added in comments to explain the  working